icu4c/source/test/intltest/loctest.cpp

```diff
@@ -3154,3 +3155,43 @@ void LocaleTest::TestBug13417VeryLongLanguageTag() {
     status.errIfFailureAndReset("\"%s\"", l.getName());
     assertEquals("equals", tag, result.c_str());
 }
+
+static const char* const tz_in_ext[] = {
```

Comment:
As these variables only are used inside of TestBug11053UnderlineTimeZone(), they wouldn't pollute the global namespace if you moved them inside of that function.